module/entityAssessors/validator/v1.js

```diff
@@ -13,6 +13,10 @@ module.exports = (req) => {
         },
         completedAssessments : function () {
             req.checkQuery('fromDate').exists().withMessage("required from date");
+        },
+        create : function () {
+            req.checkParams("_id").exists().withMessage("Required program id");
+            req.checkQuery("solutionId").exists().withMessage("Required solution id");
```

Comment:
@aman-tunerlabs - Entity is also required no?

Reply:
@aman-tunerlabs - Also user Id

Reply:
@aks30 entity is optional userId we are taking from userDetails